modguard/core/boundary.py

```diff
@@ -24,21 +24,26 @@ class BoundaryTrie:
     def __iter__(self):
         return boundary_trie_iterator(self)
 
+    @staticmethod
+    def _split_mod_path(path: str) -> list[str]:
```

Comment:
```suggestion
    def _split_mod_path(path: str, sep='.') -> list[str]:
```
if we ever want to do something like `_split_mod_path(..., "/")